docs/traffic-control.md

```diff
@@ -0,0 +1,239 @@
+# Traffic Control With Antrea
+
+## Table of Contents
+
+<!-- toc -->
+- [What is TrafficControl?](#what-is-trafficcontrol)
+- [Prerequisites](#prerequisites)
+- [The TrafficControl resource](#the-trafficcontrol-resource)
+  - [AppliedTo](#appliedto)
+  - [Direction](#direction)
+  - [Action](#action)
+  - [TargetPort](#targetport)
+  - [ReturnPort](#returnport)
+- [Examples](#examples)
+  - [Mirroring all traffic to remote analyzer](#mirroring-all-traffic-to-remote-analyzer)
+  - [Redirecting specific traffic to local analyzer](#redirecting-specific-traffic-to-local-analyzer)
+- [What's next](#whats-next)
+<!-- /toc -->
+
+## What is TrafficControl?
+
+`TrafficControl` is a CRD API that manages and manipulates the transmission of
+Pod traffic. It allows users to mirror or redirect specific traffic originating
+from specific Pods or destined to specific Pods to a local network device or a
+remote destination via various tunnels. It provides full visibility into network
+traffic, including both north-south and east-west traffic.
+
+You may be interested in using this capability if any of the following apply:
+
+- You want to monitor network traffic passing in or out of a set of Pods for
+  purposes such as troubleshooting, intrusion detection, and so on.
+
+- You want to redirect network traffic passing in or out of a set of Pods to
+  applications that analyze traffic, enforce policies, and reject traffic to
+  prevent intrusion.
+
+This guide demonstrates how to configure `TrafficControl` to achieve the above
+result.
+
+## Prerequisites
+
+TrafficControl was introduced in v1.7 as an alpha feature. A feature gate,
+`TrafficControl` must be enabled on the antrea-agent in the `antrea-config`
+ConfigMap like the following options for the feature to work:
```

Comment:
like the following options -> like the following?

Maybe it to the end of the sentence.

Reply:
Typo. I meant:

> must be enabled on the antrea-agent in the `antrea-config` ConfigMap for the feature to work, like the following: